openvdb/points/PointMove.h

```diff
@@ -186,26 +186,377 @@ class CachedDeformer
 
 namespace point_move_internal {
 
+using IndexArray = std::vector<Index>;
 
 using IndexTriple = std::tuple<LeafIndex, Index, Index>;
 using IndexTripleArray = tbb::concurrent_vector<IndexTriple>;
 using GlobalPointIndexMap = std::vector<IndexTripleArray>;
+using GlobalPointIndexIndices = std::vector<IndexArray>;
```

Comment:
All these typedefs are becoming confusing, we should really implement a class which holds these values with a sensible API to help people follow this code (specifically for GlobalPointIndexMap, GlobalPointIndexIndices and LocalPointIndexMap). Perhaps not required for this PR

Reply:
> Ultimately, it's all internal logic and not exposed via the public API, so I didn't spend too much time on it
No worries, but will help with future reviews too :)